tests/modules/text_embedding/test_embedding.py

```diff
@@ -964,6 +1013,9 @@ def test_encoding_order(loaded_model: EmbeddingModule):
         assert (
             not approx(e) == combined_vectors[i]
         ), "expected altered order to not match combined vectors"
+        assert not np.allclose(
```

Comment:
Good question.   About 10 lines up, I replaced approx() with allclose() in order to get the tests to pass for all the example truncations.  Relying on these "close" and "approx" results isn't the most reassuring but it's necessary.

Right here, this was basically a test to prove the test works and I wanted to add the allclose() test, but I didn't remove the approx() test because I didn't have to.  The both passed when there was a clear difference.

Reply:
I see - thanks for the explanations!